sympy/functions/special/error_functions.py

```diff
@@ -19,6 +19,23 @@
 # TODO series expansions
 # TODO see the "Note:" in Ei
 
+#from functools import wraps
+
+def default_var_of_integration(func):
+    #@wraps(func)
+    def _as_integral(self, t=None):
+        """
+        Return this function written as an integral, in terms of a variable
+        of integration `t`.  If `t` is omitted, a dummy variable is used.
+        """
+        if t is None:
+            t = Dummy('t')
+        return func(self, t)
+    # instead of @wraps; we want the decorator docstring
+    _as_integral.__name__ = func.__name__
+    return _as_integral
```

Comment:
I also think that the docstring should optimally come from the decorator (or, rather, from the wrapper). It is probably best to forget wraps. It should be possible to change `__name__` (and possibly `__dict__`) manually in the wrapper before returning it: `integral_wrapper.__name__ = func.__name`.